qhub/provider/cicd/github.py

```diff
@@ -0,0 +1,87 @@
+import os
+import base64
+
+import requests
+from nacl import encoding, public
+
+
+def github_request(url, method="GET", json=None):
+    GITHUB_BASE_URL = "https://api.github.com/"
+
+    for name in ("GITHUB_USERNAME", "GITHUB_TOKEN"):
```

Comment:
Can we use `GH_USERNAME`, `GH_TOKEN`? As we cannot use these in Github Secrets, the usecase for puting these on Github secrets is for Integration testing.

Reply:
Sounds good with me. Can we change the naming everywhere so that it is consistently `GH_USERNAME` and `GH_TOKEN`?

Reply:
Yeah, makes sense.

Reply:
I realise, this is not urgent as we can just map environment variables from github secret to a different key.